IrConsoleApp/Lecture5/Company.cs

```diff
@@ -0,0 +1,70 @@
+﻿using ClassLibraryHW5;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace IrynaZharnasek.Lecture5
+{
+    public class Company
+    {
+        private string companyName;
+        private Employee[] arrayOfEmployees = new Employee[100];
+
+        public void AddNewEmployee(Employee employee)
+        {
+            for (var i = 0; i < arrayOfEmployees.Length; i++)
+            {
+                if (arrayOfEmployees[i] == null)
+                {
+                    arrayOfEmployees[i] = employee;
+                    break;
+                }
+            }
+        }
```

Comment:
add empty line after method here and everywhere below

Reply:
Done